src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -154,7 +155,7 @@ public SearchQueryResultModel querySnapshot(
         WrapperQueryBuilder wrapperQuery = QueryBuilders.wrapperQuery(searchQueryRequest.getQuery());
         searchSourceBuilder.query(wrapperQuery);
 
-        var validIndexes = getValidIndexes();
+        Set<String> validIndexes = getValidIndexes();
```

Comment:
I think we should use var if we're immediately passing the variable outside the class, but if there's still logic we need to perform on it, we should keep the explicit type for readability purposes.

Reply:
http://openjdk.java.net/projects/amber/LVTIstyle.html

describes some good var uses. Like if your variable name is just a repeat of the object type. Like `var inputStream` instead of `InputStream inputStream`

Reply:
Interesting, I hadn't seen that before, I'll have to read that. The document I'd seen before is https://openjdk.java.net/projects/amber/LVTIFAQ.html